opam 2.0rc fails to fetch MetaOCaml_309_alpha_030.tar.gz when updating

```
=-=- Upgrading repositories from older opam format =-=-=-=-=-=-=-=-=-=-=-=-=-=-=
Upgrading repository "local"...
Fetching patches of ocaml-variants.3.07+1 to check their hashes...
Fetching patches of ocaml-variants.3.07+2 to check their hashes...
[ERROR] Curl failed: "/usr/bin/curl --write-out %{http_code}\\n --retry 3 --retry-delay 2 --user-agent opam/2.0.0~rc -L -o /tmp/opam-23585-09b4f7/MetaOCaml_309_alpha_030.tar.gz.part
        http://web.archive.org/web/20120209151915/www.metaocaml.org/dist/old/MetaOCaml_309_alpha_030.tar.gz" exited with code 7
[ERROR] Could not get the archive of ocaml-variants.3.09.1+metaocaml, skipping
```
Is this version really needed?

--- a/compilers/3.09.1/3.09.1+metaocaml/3.09.1+metaocaml.comp
+++ b/compilers/3.09.1/3.09.1+metaocaml/3.09.1+metaocaml.comp
@@ -1,6 +1,6 @@
 opam-version: "1"
 version: "3.09.1"
-src: "http://web.archive.org/web/20120209151915/www.metaocaml.org/dist/old/MetaOCaml_309_alpha_030.tar.gz"
+src: "https://github.com/metaocaml/metaocaml-archive/blob/master/MetaOCaml_309_alpha_030.tar.gz?raw=true"
 build: [
   ["./configure"
     "-prefix" prefix

